src/backend/package.json

```diff
@@ -9,29 +9,42 @@
   "main": "main.js",
   "author": "Digital Democracy",
   "license": "MIT",
+  "scripts": {
+    "lint": "eslint *.js \"lib/*.js\" \"test/*.js\"",
+    "test": "tape \"test/*.js\""
+  },
   "dependencies": {
     "@bugsnag/js": "^6.3.1",
     "bindings-noderify-nodejs-mobile": "^10.3.0",
-    "debug": "^4.1.1",
-    "discovery-swarm": "6.0.0",
```

Comment:
I remember this was in here to lock the version of discovery-swarm used by mapeo-core, since we were seeing errors in ^6.0.0. Do you know if the original issue was resolved? I only have vague memories of this.